app/components/preprint-form-project-select.js

```diff
@@ -55,4 +55,13 @@ export default Ember.Component.extend({
      * @property {boolean} fileSelect
      */
     fileSelect: false,
+
+    titleMatcher(node, term) {
+        // Passed into power-select component for customized searching.
+        const nodeTitle = (node.get('title') || '').toLowerCase();
```

Comment:
The default matcher for ember-power-select includes using [stripDiacritics](/cibernox/ember-power-select/blob/master/addon/utils/group-utils.js#L961-L972), which converts special characters. Since we allow those characters in project/component titles, we ought to follow that convention here as well. Here's some (untested) reference code:

```js
import {stripDiacritics} from 'ember-power-select/utils/group-utils';
```

…

```js
const sanitizedTerm = stripDiacritics(term).toLowerCase();

const fields = [
    'title',
    'root.title',
    'parent.title'
];

for (const field of fields) {
    const value = node.get(field) || '';

    if (!value)
        continue;

    const sanitizedValue = stripDiacritics(value).toLowerCase();

    if (sanitizedValue.includes(sanitizedTerm))
        return 1;
}

return -1;
```